pandas_datareader/tiingo.py

```diff
@@ -83,12 +90,15 @@ def __init__(
                 "environmental variable TIINGO_API_KEY."
             )
         self.api_key = api_key
+        self.response_format = (
```

Comment:
You should raise ValueError if not one of the expected values.

Reply:
Good point - done!